### ISSUE TITLE
Create dynamic sitemap

### ISSUE THREAD

**Issue by decabeza** [consul/consul#1170]
Create a dynamic sitemap, something similar to this file but whit dynamic domain: https://github.com/AyuntamientoMadrid/consul/blob/master/public/sitemap.xml

**Comment by microweb10** [comment]
If nobody is working on this, I can work on it. I assume you want to use something like [sitemap_generator](https://github.com/kjvarga/sitemap_generator) to dynamically generate the sitemap and something like [whenever](https://github.com/javan/whenever) to set up cron jobs to keep sitemap up to date.

**Comment by voodoorai2000** [comment]
Hello @microweb10!
Welcome back :)
Yes, that would be great, cheers!

**Comment by decabeza** [comment]
👏 👍

**Comment by microweb10** [comment]
Jejeje. Gracias! 
Perfecto! Quitarle la etiqueta de PR-welcome :)

**Comment by microweb10** [comment]
A couple of questions gentlemen:

**- To include the static pages allocated in app/views/pages on the sitemap, what would you prefer?**


Option 1: To include one by one manually.
```ruby  
  add "accessibility"
  add "census_terms"
  add "conditions"
  add "faq"
  add "general_terms"
  add "how_it_works"
  add "how_to_use"
  add "more_information"
  add "opendata"
  add "participation"
  add "participation_facts"
  add "participation_world"
  add "privacy"
  add "proposals_info"
  add "spending_proposals_info"
```

Option 2: Something more dynamic.
```ruby  
  pages = Dir.entries(File.join(Rails.root,"app","views","pages"))
  pages.each do |page|
    page_name = page.split(".").first
    add page_name if page_name.present?
  end
```

**- Do you prefer the sitemap gziped (public/sitemap.xml.gz) or not (public/sitemap.xml)?**

**Comment by voodoorai2000** [comment]
+1 to option 2 and not gziped

**Comment by voodoorai2000** [comment]
Closed via #1278
